.env.local

```diff
@@ -1,4 +1,4 @@
 SQL_USER=postgres
 SQL_NAME=openousd
 SQL_PASSWORD=openousd
-SQL_HOST=localhost
+SQL_HOST=postgres
```

Comment:
from within the node/api container, this is the host name for the postgres container